python-main.js

```diff
@@ -588,10 +588,16 @@ function web_editor(config) {
             }
         });
         // Bind the ESCAPE key.
+        var button = $('#command-download');
+        var otherButton = $('#command-flash');
```

Comment:
These variables are created here but never used?

Reply:
whoops. a previous experiment that I left in :) Have now removed